Utils/src/GenParticlesProducer.cc

```diff
@@ -186,6 +186,10 @@ enum particle_type
     dm1=51,
     dm2=52,
     dm3=53,
+    
```

Comment:
Remove this space. You should always try to keep the same code style for a file you are updating.